Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs

```diff
@@ -234,6 +234,7 @@ public async Task Should_be_able_to_create_all_the_definitions_in_a_policy()
         const long maxLengthBytes = 5000;
         const Overflow overflow = Overflow.RejectPublish;
         uint? consumerTimeout = fixture.RabbitmqVersion >= new Version("3.12") ? 3600000 : null;
+        Dictionary<string, object> extensionData = new Dictionary<string, object> { { "max-in-memory-length", 1000000 } };
```

Comment:
I'm a bit ambivalent, I like adding the extension data to every contract to minimize the changes to the library with new versions, but also I think it's a good idea to add strongly typed properties if we know them to exist in the current versions, what do you think @Pliner ?

Reply:
@zidad sure, `JsonExtensionData` is not a replacement for any existing strongly typed property. The more well-known properties are strongly typed, the better. No agrue about it.

However, it seems the project doesn't have enough resources to maintain all the records aligned to actual RabbitMQ HTTP API. And honestly, some of them are used too rarely to justify maintaining effort (e.g. `Overview`->`Listener`->`SocketOpts`).